wrapper/src/main/java/software/amazon/jdbc/plugin/efm/MonitorImpl.java

```diff
@@ -90,6 +94,7 @@ public MonitorImpl(
     this.monitorService = monitorService;
 
     this.contextLastUsedTimestampNano = this.getCurrentTimeNano();
+    setMonitoringEndpoint();
```

Comment:
Convert it to function?
this.monitoringEndpoint = getMonitoringEndpoint(this.hostSpec)